src/pip/_internal/req/req_install.py

```diff
@@ -753,8 +751,11 @@ def install_editable(
             # FIXME: should we do --install-headers here too?
             with self.build_env:
                 call_subprocess(
-                    make_setuptools_shim_args(self.setup_py_path) +
-                    list(global_options) +
+                    make_setuptools_shim_args(
```

Comment:
Since this is getting a bit big for an argument evaluation, I would evaluate the return value of `make_setuptools_shim_args()` on its own line before calling `call_subprocess()`. Can it be done outside the `self.build_env` and `indent_log()` context managers? That would be best for indentation reasons.

Reply:
`self.build_env` kind of modifies the environment before running the command, let me have good look at it once later today and confirm if that's fine.